blog/README.md

```diff
@@ -0,0 +1,298 @@
+# CoreOS Layering
+
+In this blog we are going to explore the coreos-layering technology in order to
+inject kernel-modules to an OpenShift node.
+
+We will use the [driver-toolkit](https://github.com/openshift/driver-toolkit)
+as a base image in order to build the [simple-kmod](https://github.com/openshift-psap/simple-kmod)
+kernel module and then add the `.ko` to a new layer on top of the RHEL-CoreOS
+image that runs on the node.
+
+### Different methods of injecting kernel modules to a cluster
+
+* Day 0: Creating a custom RHCOS image for deployment including the kernel module
+* Day 2 MCO: Using the Machine Config Operator with a layered image (what we are showing in this blog)
```

Comment:
I would suggest Day 1/2 as MCO can be used for either.

Reply:
Done.